cmd/eudico-stats/main.go

```diff
@@ -115,24 +124,54 @@ var runCmd = &cli.Command{
 		genesisTime := time.Unix(int64(gtp.MinTimestamp()), 0)
 
 		go func() {
-			t := time.NewTicker(time.Second)
+			// trigger calculation every 30 seconds
```

Comment:
It is actually doing it every second right?

This approach is OK for the current metric, but we should consider the `EventListener` approach so that we have fine-grained data for others.

Reply:
I can delay this to every 30 seconds.
Yeah, I tried setup the event listener, but probably something I did wrongly. But the `lotus-stats` is also doing sth similar for some of the metrics I think.

Reply:
This is fine, I just asked because the comment and the code don't match.

Reply:
And the suggestion of collecting additional metrics is to get a sense of the processes we need in place before committing a lot of time to an approach that doesn't work and having to revert this after merging.